### ISSUE TITLE
Control of rendered font size

### ISSUE THREAD

**Issue by marcusps** [uranusjr/macdown#482]
Currently the preferences allow for control of the editor's font (face and size), but it does not allow for any control of the font size on the pane for rendered text.

It would be nice to be able to scale the text, at the very least.

**Comment by Jmuccigr** [comment]
This would better be the kind of thing in a Toolbar, no?

**Comment by marcusps** [comment]
I am not so sure a toolbar would be necessary.  Having it on the menu or in
the preferences would be fine by me — I don't expect to change it that
often.

**Comment by Jmuccigr** [comment]
No, not necessary, but this is just the kind of thing one usually finds on the toolbar.

**Comment by uranusjr** [comment]
Not directly related, but there’s a “Scale preview based on editor font size” preference. If you tick that, and increase the editor font size, the preview font will be resized automatically.

**Comment by tbredin** [comment]
@font-face support or even just grabbing a system font for the specified CSS font stack would be pretty great, right?

**Comment by uranusjr** [comment]
If you are going in implementing it :)

**Comment by ssstonebraker** [comment]
@shacker i know i am almost eight years late but i just submitted a pull request for your Cmd+/Cmd- functionality
https://github.com/MacDownApp/macdown/pull/1364
